client/src/shared/state/GUI.js

```diff
@@ -59,14 +59,14 @@ class GUI {
          * @todo find somewhere else for this, doesn't really belong with this GUI stuff
          * @type {Number}
          */
-        this.musicVolume = 0;
+        this.musicVolume = 50;
```

Comment:
😆  I would probably amend the commit that you initially changed this in so that there is no history of someone changing the volume, to keep the master history clean

Reply:
Looks like upstream squashes commits when they merge anyway, so this might not matter